### ISSUE TITLE
Not able to open old Wiki and nor able to open forum

### ISSUE THREAD

**Issue by martin-ac** [navit-gps/navit#1227]
Hello everybody,
for sure I would do what is asked for above, but currently and at least in the past 7 days I'm not - and I think nobody is - able to access
- the old Navit wiki at the above given address or to open https://wiki.navit-project.org
- same with the forum at https://forum.navit-project.org/
- BUT !!! am able to open https://www.navit-project.org/ without any problem or error.

I already wrote emails about this to four project managers but didn't get answers nor did this change anything. But maybe this is because the used mail domain is 
@navit-project.org which may not be working correct too?

I tried with current versions of firefox and chromium and 4 different pc's / nb's running Kubuntu 20.04 and 22.04 where I do not have any other problem with other pages in the internet and so am pretty sure there is an error with the above mentioned pages or servers hosting them.

Could someone please check and give me a feedback?
Thanks in advance!
Martin

**Comment by jkoan** [comment]
Hi @martin-ac 
Cc @navit-gps/navit 
TL;DR: The server which hosted some of our infrastructure was taken offline. This server was provided by fosshost. Fosshost is closing down. Data is backed up and will be migrated.

Long Form:
Our Hoster for the navit server has shut down our server on February the 10th. I took a backup of all data. Now we still need to migrate all the relevant data, most important wiki.

We basically have the following services:
nginx (http frontend server) -> not needed any more
matterbridge -> can probably be hostet by me personally as I also host the IRC logs
forum -> discontinue, maybe achive it in some form (internet achive; archive.org?). Use GitHub discussions, GitHub issues?
cloud -> Not need anymore, only used for Fossdem Präsentation 
trac -> migrate to github issues
wiki -> migrate all content to Read the Docs #425 also see #1195 . Open: What about old links: Add a github pages site with mappings, and point wiki.navit-project to it? Alternatively less future proof export static version of the wiki and put on github pages?
ldap_gw -> not needed anymore 
backend (DB) -> definetly take dumps, but except from that also not needed anymore
ses (Mail Gateway) -> not needed anymore, was broken anyway 
www -> already on GitHub pages, so no problems 

For email I think this is still hosted on the even older servers which some of the initial team members set up. I never had access to those servers so I can't help with that. But yes has definitely nothing to do with that.

**Comment by OLFDB** [comment]
Meanwhile you can use the [webarchive](https://web.archive.org/web/20230202050307/https://wiki.navit-project.org/index.php/Main_Page).

**Comment by jkoan** [comment]
Or the WIP prototype for read the Doc's here https://navit--1195.org.readthedocs.build/en/1195/

Feedback appreciated :)

**Comment by freecycler** [comment]
@martin-ac
 I found similarly. Even before that, when the site certificate was expired I had first posted a message on the forums, then sent PMs to the moderators, then finally sent email to the 4 about the certificate on 1/7. Later the website was down and returning SQL errors so 1/20 I sent another email to those same 4 addresses, getting similar non-results. A different SQL error prompted a third email 1/28. Still later the forums were down completely, not even a SQL error, browsers just timeout and say connection refused, so I sent a fourth email on 2/21, still no results. I guess what jkoan says is the reason why.

@jkoan

> forum -> discontinue, maybe achive it in some form (internet achive; archive.org?). Use GitHub discussions, GitHub issues?

This can be somewhat problematic. The forums were the best place for user level help in cases where the wiki(s) did not cover the question at hand (and that is assuming that users even know they exist), but we do not yet know whether it is actually a bug vs something we just need better understanding of intended use.

Even knowing the IRC channel existed, it took me more than a month to actually find its archive location. Even then I found there were only a couple people/bots logged in, and repeatedly got no response to my inquiries. Looking through the logs since 1 Jan, I see similarly for others; there were a few people who came to ask questions and in one case offer to help w/development but nobody got any response. I think in general IRC is not well suited to such use since it requires participants to both be logged in and at their computers - and even paying attention - at the same time, not so easy if we are all in different time zones. And the archives are not so easy to read and search as the forums were, so unless both parties are online at the same time, any real discussion is basically impossible there.

At the very least I would ask if you could find some way to put the old forums content back on line even if they are archived as read only because the material they contained was very helpful.

As for going forward, I think many less technical users would be put off from asking for help in the github issues list since they are not yet trying to report a bug until they can first ask how it is intended to work or if there is some user error they want to learn to fix. To some extent github discussions would have a similar impression problem, plus its existence is even less commonly known among nontechnical users.

While it is good to see the new wiki being populated and content added, they are not really a place for interactive discussions.

There is still a need for somewhere to have those discussions, that are easy for newcomers to find and use; the forums used to fill that need. So please, when time permits, bring back the forums!
